build.sbt

```diff
@@ -1,5 +1,6 @@
 val common = Seq(
-  scalaVersion := "2.12.10",
+  ThisBuild / classLoaderLayeringStrategy := ClassLoaderLayeringStrategy.ScalaLibrary,
```

Comment:
what is this :o

Reply:
Oh - I ran out of MetaSpace running the tests and sbt printed a message suggesting that adding this might help it not run out of memory last time. I don't actually know if it makes any difference

Reply:
fair enough! :+1: